react-lists/my-app/src/PokemonList.tsx

```diff
@@ -0,0 +1,29 @@
+type Props = {
```

Comment:
You have this in two places. If you change your mind and decide to change it (for example, making `number` a number instead of a string), then you have to remember to change it here and in `App`. You should declare it once and export it. Then you can import it in the other places you use it.